python/sglang/srt/utils.py

```diff
@@ -1001,42 +1001,34 @@ def point_to_point_pyobj(
     dst: int = 1,
 ):
     """Send data from src to dst in group using DeviceToDevice communication."""
-
+    device = get_device()
```

Comment:
Could we make this method and `broadcast_pyobj` accept device_type as parameter?